src/main/components/MobileNav.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {Action} from "./Actions";
+import {DiscoverIcon, ProfileIcon, SocialIcons, WalletIcon, XIcon} from "../static/icons/Icons";
+
+const MobileNav = ({open, onOpenChange}) => {
+  if(!open) { return null; }
+
+  return (
+    <div className="mobile-nav__menu">
+      <Action icon={XIcon} className="dark mobile-nav__close-button mobile" onClick={() => onOpenChange(false)} />
+      <div className="mobile-nav__menu-content">
+        <div className="mobile-nav__menu-header mobile-nav__menu-section">
+          <Action className="mobile-nav__menu-header-item" icon={DiscoverIcon} alt="Projects" to="/about">
+            <span>Projects</span>
```

Comment:
is there a reason for having additional <span>s here?

Reply:
I was styling the font size but I moved that to the parent and removed the span. I think I had more css for that span before.